geomesa-web/geomesa-web-core/src/main/scala/org/locationtech/geomesa/web/core/SpringScalatraBootstrap.scala

```diff
@@ -30,9 +30,12 @@ trait GeoMesaScalatraServlet extends ScalatraServlet with LazyLogging {
 
   // This may be causing issues within scalatra, to paraphrase a comment: "Wrapped requests are probably wrapped for a reason."
   // https://geomesa.atlassian.net/browse/GEOMESA-1062
```

Comment:
Not sure if we should close that old ticket - I haven't tested file upload support but since we're not unwrapping the request it might work now.